packages/react/src/date-picker-next/DatePickerNext.tsx

```diff
@@ -241,23 +237,13 @@ export const DatePickerNext = ({
 				value={inputValue}
 			/>
 			<CalendarProvider yearRange={yearRange}>
-				{/* We duplicate the Popover + Calendar as a workaround for a bug that scrolls the page to the top on initial open of the calandar - https://github.com/gpbl/react-day-picker/discussions/2059 */}
```

Comment:
I don't think this has been fixed so shouldn't have been removed?

Reply:
Interesting that this one caused a scroll up, when it fixed it v8 😅 . I think we'll need to do some testing